toolchain/check/testdata/basics/no_prelude/multi_error.carbon

```diff
@@ -43,9 +43,9 @@ fn Boo[a: ()]() {}
 // CHECK:STDOUT:   %Boo.decl: %Boo.type = fn_decl @Boo [concrete = constants.%Boo] {} {}
 // CHECK:STDOUT: }
 // CHECK:STDOUT:
-// CHECK:STDOUT: fn @Foo[<error>: <error>]();
+// CHECK:STDOUT: fn @Foo();
```

Comment:
Dropping errors from the output seems a bit confusing, can they be preserved?

Reply:
If the absence of the errors is a bug, it's not a new bug here, it's a pre-existing bug on line 42, which doesn't show any errors in the declaration of `Foo`.